notebooks/charactercodex_llamafile.ipynb

```diff
@@ -0,0 +1,2607 @@
+{
```

Comment:
<p>Might make sense to mention mozilla somewhere here, llamafile is a project by mozilla that simplifies...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>done</p>